coord/src/rmr/rmr.c

```diff
@@ -408,7 +408,7 @@ int MR_Fanout(struct MRCtx *mrctx, MRReduceFunc reducer, MRCommand cmd, bool blo
     mrctx->bc = RedisModule_BlockClient(
         mrctx->redisCtx, unblockHandler, timeoutHandler,
         redisMajorVesion < 5 ? (void (*)(RedisModuleCtx *, void *))freePrivDataCB : freePrivDataCB_V5,
-        timeout_g);
+        0); // timeout_g);
```

Comment:
Explain?

Reply:
see my [reply here](https://github.com/RediSearch/RediSearch/pull/3853#discussion_r1349698820). It was also requested by @DvirDukhan so it is clear that this is what currently happening. We can discuss it in person.